test/e2e/main/main_test.go

```diff
@@ -303,6 +287,33 @@ func TestNodeWithEphemeralStorage(t *testing.T) {
 	}
 }
 
+// Test if single node with a storage class
+func TestNodeWithStorageClass(t *testing.T) {
+	t.Parallel()
+	kindDefaultStorageClass := "standard"
```

Comment:
This shouldn't work outside the Kind, however QA's are testing on non-Kind envs.
You could adopt this script to get default StorageClass and then test again it.

Reply:
kk, want me to add a [local storage class](https://kubernetes.io/docs/concepts/storage/storage-classes/#local) to the cluster in case the get default `StorageClass` returns nil? Or could we assume that we'll always have a default `StorageClass` in QA?

Reply:
It's probably you need to get a StorageClass with `storageclass.kubernetes.io/is-default-class: "true"` annotation and play around it. This should work on any platform including OCP/OKD/Kind/etc.

Reply:
Indeed, I just got the `StorageClass` name of the one with the `storageclass.kubernetes.io/is-default-class: "true"` annotation, as suggested. This should now be compatible with any k8s-powered platform (see [k8s official docs](https://kubernetes.io/docs/tasks/administer-cluster/change-default-storage-class/)), thanks!